pages/platform/kubernetes-k8s/backup-and-restore-cluster-namespace-and-applications-with-trilio/assets/manifests/triliovault-manager-v2.6.4.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: triliovault.trilio.io/v1
+kind: TrilioVaultManager
+metadata:
+  labels:
+    triliovault: k8s
+  name: tvk
+spec:
+  trilioVaultAppVersion: 2.6.4
```

Comment:
The appversion should not be defined here, or add a warning to check it before (the actual version is 2.6.6)

Reply:
Removed this yaml manifest in the latest release. The `TVM` installation is handled directly through a single helm chart of `triliovault-operator` making is a one-click install.